### ISSUE TITLE
Web interface should display version information

### ISSUE THREAD

**Issue by Bjwebb** [Bjwebb/bookish#3]
Possibly including github commit hash and docker image id.

**Comment by caprenter** [comment]
@Bjwebb can this be closed now?

**Comment by Bjwebb** [comment]
No, I've added the github commit hash to the footer, but I'm not yet showing the anything relating to the docker image.
